app/cognito/utils/user.py

```diff
@@ -37,7 +37,7 @@ def delete_user(user: User) -> bool:
     return deleted
 
 
-def update_user(user: User) -> bool:
```

Comment:
We should also change the log from `warning` to `critical`.

Reply:
Right, will fix - thanks!